src/portable/portable.c

```diff
@@ -134,98 +134,6 @@ int portable_metadata_hashmap_to_sorted_array(Hashmap *unit_files, PortableMetad
         return 0;
 }
 
-static int send_item(
-                int socket_fd,
-                const char *name,
-                int fd) {
```

Comment:
ah, good catch, that this can be reused.

that said, why not keep this helper, and make it just a wrapper around send_one_fd_iov() but do the copy_data_fd() first? i.e. the code using this does the same thing twice otherwise